core/templates/pages/topic-viewer-page/info-tab/topic-info-tab.directive.ts

```diff
@@ -38,11 +48,15 @@ angular.module('oppia').directive('topicInfoTab', ['UrlInterpolationService',
         'WindowDimensionsService', function(WindowDimensionsService) {
           var ctrl = this;
 
-          ctrl.$onInit = function() {
-            ctrl.isMobile = false;
+          ctrl.checkSmallScreenWidth = function() {
             if (WindowDimensionsService.getWidth() <= 1024) {
-              ctrl.isMobile = true;
+              return true;
             }
+            return false;
```

Comment:
Can you change this to:

`return WindowDimensionsService.getWidth() <= 1024;`

Reply:
@aks681

Reply:
This change should go in #9362 @aks681